packages/plugin-loansresearch-api/src/graphql/resolvers/queries/loanResearch.ts

```
@@ -34,8 +35,31 @@ const lsQueries = {
+      defaultValue: [],
+    });
+
+    const findResearch = await models.LoansResearch.findOne({ dealId });
+
+    if (findResearch) {
+      await models.LoansResearch.updateOne(
+        { dealId },
+        {
+          $set: {
+            increaseMonthlyPaymentAmount:
+              (loansContracts && loansContracts?.firstSchedules[0]?.total) || 0,
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)